assets/server/popup.html

```diff
@@ -25,7 +25,11 @@ <h1 class="title">{{ .PageTitle }}</h1>
           <select class="content-select" id="category" name="category">
             {{ range $element, $value := .Content.Categories }}
             {{ if eq $element $context.Category }}
-            <option value="{{ $element }}" selected="selected">{{ $element | toTitle }}</option>
+            {{ $hint := $value.Hint }}
+            {{ if or (not $hint) (eq $hint "") }}
+              {{ $hint = "i.e. issue/xxxxx" }}
+            {{ end }}
+            <option value="{{ $element }}" selected="selected" hint="{{ $hint }}">{{ $element | toTitle }}</option>
```

Comment:
`not $hint` should cover the empty string case too, thus this can be simplified:

```suggestion
            <option value="{{ $element }}" selected="selected" hint="{{ or $value.Hint "i.e. issue/xxxx" }}">{{ $element | toTitle }}</option>
```

However, I feel like we should _always_ set a default hint in the controller to avoid this logic in the template.

Reply:
I agree. All defaults should come from https://github.com/abcxyz/jvs/blob/main/apis/v0/plugin.go#L74-L79

Reply:
To clarify, I think we should just remove this special hint logic. When a category is selected, its corresponding hint should show up.

Reply:
removed this logic in the latest version.